asilib/asi/themis.py

```diff
@@ -135,21 +139,27 @@ def themis(
     _skymap = themis_skymap(location_code, _time, redownload)
 
     if custom_alt==False:
-        alt_index = np.where(_skymap['FULL_MAP_ALTITUDE'] / 1000 == alt)[0] #Compares the altitudes versus the ones provided by default and chooses the correct index that correlates to the chosen alitudes
+        alt_index = np.where(_skymap['FULL_MAP_ALTITUDE'] / 1000 == alt)[0]
         assert (
             len(alt_index) == 1
         ), f'{alt} km is not in the valid skymap altitudes: {_skymap["FULL_MAP_ALTITUDE"]/1000} km. If you want a custom altitude with less percision, please use the custom_alt keyword'
         alt_index = alt_index[0]
-        lat=_skymap['FULL_MAP_LATITUDE'][alt_index, :, :] #selects lat lon coordinates from data provided in skymap
+        lat=_skymap['FULL_MAP_LATITUDE'][alt_index, :, :]
         lon=_skymap['FULL_MAP_LONGITUDE'][alt_index, :, :]
-    else:
-        lat,lon = asilib.skymap.geodetic_skymap( #Spherical projection for lat lon coordinates
+    elif custom_alt =='geodetic':
```

Comment:
I'd like this code to work if the user supplied "Geodetic" (or "Interp"). Would you mind changing these lines to:

```suggestion
    elif isintance(custom_alt, str) and (custom_alt.lower() =='geodetic'):
```